Library/Formula/mplayer.rb

```diff
@@ -8,9 +8,11 @@ class Mplayer < Formula
   end
 
   bottle do
-    sha256 "5504434dce05f51d0d6336426f64af559740203193f6f34d1ef250ebfe955d8f" => :el_capitan
-    sha256 "6d6ccf0b963bc4a0edd48bc7d1e1008db888dc38b91fe85c6c58c5d6dded4f48" => :yosemite
-    sha256 "d90208d62889b8c74d06917ba926f91fddb039ab15b27c825e4356b75b9d105b" => :mavericks
+    revision 1
```

Comment:
I thought that too, and I myself never entered that and wondered where that came from. I just followed the instructions https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/How-To-Open-a-Homebrew-Pull-Request-%28and-get-it-merged%29.md#how-to-open-a-homebrew-pull-request-and-get-it-merged but it is very possible I did something wrong